### PR TITLE
feat: add more assertion commands

### PR DESCRIPTION
**Pull request by justinkambic**
Resolves https://github.com/elastic/synthetics-recorder/issues/20.

~This is WIP -~ adding additional assertion commands. Waiting for agent-side updates to synthetics before building this out further.

### CODE REVIEW

**Comment by justinkambic** [comment]
Any additional change requests before this is ready to go?

File: src/common/shared.ts [hunk 1]
```diff
@@ -5,16 +5,36 @@ const { ipcRenderer: ipc } = window.require("electron-better-ipc");
...
+    text: "Check Hidden",
   },
   {
     value: "isVisible",
     text: "Check Visibility",
   },
   {
-    value: "isHidden",
-    text: "Check Hidden",
+    value: "innerText",
```
Comment (vigneshshanmugam) [inline src/common/shared.ts:16]: nit: keep innertext and textContent close to one another and other boolean checks before.

**Review by vigneshshanmugam — APPROVED** [review]
Didnt run it locally, But code LGTM

File: src/common/shared.ts (hunk 1 above)
Comment (justinkambic) [reply src/common/shared.ts:16]: Will do.

**Pull request merged**